### PR TITLE
13.0 mig project role

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]
@alfadil please follow https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch to separate formatting commit from actual migration and squash the administrative commits.

**Comment by ghost** [comment]
> @alfadil please follow https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-13.0#technical-method-to-migrate-a-module-from-120-to-130-branch to separate formatting commit from actual migration and squash the administrative commits.

Thanks for your review , Please check my fix.

**Comment by alexey-pelykh** [comment]
Please fix the commit message:
image

**Comment by ghost** [comment]
> Please fix the commit message:
> image

done :)

**Review by dreispt — APPROVED** [review]
LGTM

**Review by HaraldPanten — APPROVED** [review]
Functional review 👍

**Review by alexey-pelykh — APPROVED** [review]

File: project_role/models/project_assignment.py [hunk 1]
```diff
@@ -15,7 +15,7 @@ class ProjectAssignment(models.Model):
     company_id = fields.Many2one(
         comodel_name="res.company",
         string="Company",
-        default=lambda self: self.env["res.company"]._company_default_get(),
+        default=lambda self: self.env.user.company_id,
```
Comment (alexey-pelykh) [inline project_role/models/project_assignment.py]: Use `self.env.company`, that's the v13 way it seems

**Review by alexey-pelykh — CHANGES_REQUESTED** [review]

**Review by alexey-pelykh — APPROVED** [review]

**Pull request merged**